pandas_sphinx_theme/bootstrap_html_translator.py

```diff
@@ -62,3 +62,10 @@ def visit_table(self, node):
         #     classes.append('align-%s' % node['align'])
         tag = self.starttag(node, "table", CLASS=" ".join(classes))
         self.body.append(tag)
+
+    def visit_container(self, node):
+        """
+        Remove 'docutils' and 'container' classes to overcome interference
+        between docutils container class and bootstrap container class
+        """
+        self.body.append(self.starttag(node, "div", CLASS=""))
```

Comment:
couldn't this just be something like below? I think the details probably are off, but the general idea?
```suggestion
        classes = ' '.join(class for class in node.attributes.classes if class != "container")
        self.body.append(self.starttag(node, "div", CLASS=classes))
```

Reply:
Yes, that's indeed more correct (because the whole purpose was to use eg `.. container:: some-bootstrap-class`, so we indeed need to pass that through)

But, that still doesn't solve the issue of removing "container" causing issues for projects that rely on it for theming (like nbsphinx)

Reply:
yeah that makes sense - probably not a good idea to alter fundamental default sphinx behavior

Reply:
Actually, I was a bit fast with commenting, as there *should* be some other way that the user-defined classes are pass through, as the base docutils implementation is:

```

    def visit_container(self, node):
        self.body.append(self.starttag(node, 'div', CLASS='docutils container'))
```

so we have exactly the same just without the two classes. I suppose the CLASS option above is only additive to the classes that are already defined in the node itself (coming from parsing the `.. container::` directive)

Reply:
Indeed, as far as I understood, the other classes are passed as well, this removes only  these two. Still, there is probably a better way to do this, but I do not have any other idea for the moment.